stories.json

```diff
@@ -31,7 +31,97 @@
   },
   {
     "permalink": "good-times",
-    "text": "We have a holistic view of people. Work life, a family life,  includeanything life are not silos. People have one life. Organizations need to consider that. It is less about partying and onsites (even if they mean a lot to me <3), it is more about having a purpose in your daily work (no bullshit jobs) and the `go` to self-organize. It is about no boundaries to thinking and getting the information you need to make decisions (transparency) and decide as an adult person what is best for you. It is about the flexibility to integrate all elements of life into your day.",
+    "text": "We have a holistic view of people. Work life, a family life,  include anything life are not silos. People have one life. Organizations need to consider that. It is less about partying and onsites (even if they mean a lot to me <3), it is more about having a purpose in your daily work (no bullshit jobs) and the `go` to self-organize. It is about no boundaries to thinking and getting the information you need to make decisions (transparency) and decide as an adult person what is best for you. It is about the flexibility to integrate all elements of life into your day.",
+    "background": "northernlights"
+  },
+  {
+    "permalink": "trust",
+    "text": "Trust is given from the start and your team really does trust you to complete the work to the best of your ability. This means that there's also a lot of freedom and opportunities to be creative as you're able to interpret and deliver work without strict hardline 'this is how we do it'.",
+    "background": "northernlights"
+  },
+  {
+    "permalink": "startup",
+    "text": "When I recruit, I always say 'We are an old startup'. We are not a bunch of graduates who do internships. We are experienced but have not reached the point, where 'I made it always like that' is an option. We are not motivated by being a 'fancy Berlin style' startup. (Sorry, Berlin!). We do not need fruits and a soccer table. Our Founders have done this before and we all know how to work. We do not save money when it comes to salaries, equipment, external advise. We follow the rules when they have to be followed.",
+    "background": "northernlights"
+  },
+  {
+    "permalink": "pull-culture",
+    "text": "My job at Giant Swarm focuses on recruiting and employer branding. The first part is pretty straightforward but employer branding has many different aspects to it. For the initial time I was waiting for someone to tell me what they had in mind and what they expected from me. It took me quite a while (also because recruiting initially took up most of my time) and some clear words from Oli to realise that they’ve been waiting for me to come up with ideas. This made the pull-culture pretty obvious for me. Noone would tell me what to do but instead I had to tell them what I wanted and needed.",
```

Comment:
```suggestion
    "text": "My job at Giant Swarm focuses on recruiting and employer branding. The first part is pretty straightforward but employer branding has many different aspects to it. Initially, I was waiting for someone to tell me what they had in mind and what they expected from me. It took me quite a while (also because recruiting initially took up most of my time) and some clear words from Oli to realise that they’ve been waiting for me to come up with ideas. This made the pull-culture pretty obvious for me. Noone would tell me what to do but instead I had to tell them what I wanted and needed.",
```